src/compiler/irgen.jl

```diff
@@ -238,10 +332,11 @@ function lower_throw!(mod::LLVM.Module)
                 changed = true
             end
 
-            @compiler_assert isempty(uses(f)) ctx
+            @compiler_assert isempty(uses(f)) job
          end
      end
```

Comment:
Extra whitespace before this `end` (not your fault, I'm just marking it for one of us to fix).